src/ActorMultiVertex.cpp

```diff
@@ -984,13 +1006,16 @@ class LunaActorMultiVertex: public Luna<ActorMultiVertex>
 		SET_SIDE(4, rect.bottom);
 		lua_pop(L, 1);
 		SET_SIDE(2, delay);
-		const float width_ratio= tex->GetImageToTexCoordsRatioX();
-		const float height_ratio= tex->GetImageToTexCoordsRatioY();
-		state.rect.left= state.rect.left * width_ratio;
-		state.rect.top= state.rect.top * height_ratio;
-		// Pixel centers are at .5, so add an extra pixel to the size to adjust.
-		state.rect.right= (state.rect.right * width_ratio) + width_ratio;
-		state.rect.bottom= (state.rect.bottom * height_ratio) + height_ratio;
+		if (tex != nullptr)
```

Comment:
This nullptr check is not needed. The luaL_error above returns which handles the nullpr check already.

Reply:
Thanks for noting this!

Reply:
Btw if you fixed this, there's no follow up commit.

Reply:
Sorry, just getting it done now.